src/announcements.py

```
@@ -214,3 +215,52 @@ async def send_announcement(
+        logger.warning("Developer guild id %s not found or bot not in guild.", dev_guild_id)
+        return
+
+    try:
+        channel = await get_admin_channel(guild)
+        body = message
+        if mention_user_id:
+            body = f"<@{mention_user_id}> {message}"
+        await channel.send(body)
+        logger.info("Sent admin update to guild %s channel %s", guild.id, channel.name)
+    except Exception as e:
+        logger.exception("Failed sending admin update: %s", e)
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)